### PR TITLE
Update intro-to-curve-dodao-academy.yaml

### PR DESCRIPTION
**Pull request by gregerodrigues**

### CODE REVIEW

File: src/intro-to-curve-dodao-academy.yaml [hunk 1]
```diff
@@ -96,7 +96,7 @@ steps:
             key: c3022c38
           - content: 'By creating tripool '
             key: 9f74db07
-        content: How does curve finance overcome the problem of slippage
+        content: How does curve finance overcome the problem of slippage that occurs due to volatality, while order being executed ?
```
Comment (RobinNagpal) [inline src/intro-to-curve-dodao-academy.yaml:99]: Slippage is not related to gauge weights. Gauge weights are for rewards

File: src/intro-to-curve-dodao-academy.yaml [hunk 2]
```diff
@@ -145,31 +145,31 @@ steps:
           - 52c4c10a
           - 31cbd68f
         choices:
-          - content: To earn rewards through bribing
+          - content: To earn new tokens for veCRV wotes in bribing
```
Comment (RobinNagpal) [inline src/intro-to-curve-dodao-academy.yaml:148]:
```suggestion
          - content: To earn new tokens for veCRV votes in bribing
```

File: src/intro-to-curve-dodao-academy.yaml [hunk 3]
```diff
@@ -145,31 +145,31 @@ steps:
...
             key: ad0cd11f
           - content: To get voting power in gauge weight
             key: c38ae3e6
           - content: Participate in Curve DAO governance
             key: 52c4c10a
-          - content: Gain more rewards by investing veCRV in Convex Finance
+          - content: To earn new tokens for veCRV wotes in bribing
```
Comment (RobinNagpal) [inline src/intro-to-curve-dodao-academy.yaml:154]: The first and last option is the same. You can make it a single choice question with the last choice as "All the above"

File: src/intro-to-curve-dodao-academy.yaml [hunk 4]
```diff
@@ -145,31 +145,31 @@ steps:
...
           - '016e3941'
           - 4eb52fb4
-          - 057f3dd6
+  
         choices:
-          - content: You are given veCRV for locking CRV token
+          - content: They get more governance power over other
             key: 057f3dd6
-          - content: You can withdraw staked CRV any time
+          - content: They get o.1 USDC for every dollar locked
```
Comment (RobinNagpal) [inline src/intro-to-curve-dodao-academy.yaml:166]:
```suggestion
          - content: They get 0.1 USDC for every dollar locked
```

File: src/intro-to-curve-dodao-academy.yaml (hunk 3 above)
Comment (gregerodrigues) [reply src/intro-to-curve-dodao-academy.yaml:154]: ok sir

File: src/intro-to-curve-dodao-academy.yaml [hunk 5]
```diff
@@ -145,31 +145,31 @@ steps:
...
             key: c38ae3e6
           - content: Participate in Curve DAO governance
             key: 52c4c10a
-          - content: Gain more rewards by investing veCRV in Convex Finance
+          - content: To earn new tokens for veCRV wotes in bribing
             key: 31cbd68f
         content: 'Advantages of owning veCRV:'
         type: MultipleChoice
         uuid: 6f7478f3-c3d7-43fc-98f6-61a661c87f31
       - answerKeys:
```
Comment (RobinNagpal) [inline src/intro-to-curve-dodao-academy.yaml:159]: 057f3dd6 is a correct choice

File: src/intro-to-curve-dodao-academy.yaml [hunk 6]
```diff
@@ -250,23 +250,23 @@ steps:
...
+          - content:   You get additional veCRV
             key: 5a79c680
-          - content: They are given different token
+          - content:   You get additional veCRV
             key: a2e59bb3
-          - content: They are get more boost in their rewards
+          - content: You get more boost in your rewards
             key: 47b3410d
-          - content: Rewarded with CRV token
+          - content:You get more boost in your rewards
```
Comment (RobinNagpal) [inline src/intro-to-curve-dodao-academy.yaml:267]: Option 3 and 4 is the same

File: src/intro-to-curve-dodao-academy.yaml [hunk 7]
```diff
@@ -250,23 +250,23 @@ steps:
...
         type: MultipleChoice
         uuid: 02502c1d-f77b-43c3-bfd8-5b89d0d03ef9
       - answerKeys:
           - a2e59bb3
         choices:
-          - content: They get additional veCRV
+          - content:   You get additional veCRV
             key: 5a79c680
-          - content: They are given different token
+          - content:   You get additional veCRV
```
Comment (RobinNagpal) [inline src/intro-to-curve-dodao-academy.yaml:263]: Option 1 & 2 is the same

File: src/intro-to-curve-dodao-academy.yaml [hunk 8]
```diff
@@ -314,15 +314,15 @@ steps:
...
         choices:
-          - content: Non user friendly interface
+          - content: Complicated steps involved,which makes staking difficult for new user
             key: 59fc15ea
           - content: Gauge votes are influenced by bribes
             key: cc268588
           - content: Uncontrollable slippage
             key: ec62345e
-          - content: veCRV tokens cannot be moved to different wallets
+          - content: veCRV tokens dont remain liquid once locked
```
Comment (RobinNagpal) [inline src/intro-to-curve-dodao-academy.yaml:323]:
```suggestion
          - content: veCRV tokens don't remain liquid as they are locked for a time period
```

File: src/intro-to-curve-dodao-academy.yaml (hunk 1 above)
Comment (gregerodrigues) [reply src/intro-to-curve-dodao-academy.yaml:99]: yes sir, here the answer is 2nd option.

**Pull request closed without merging**